### DISCUSSION TITLE
Menu organization

### DISCUSSION THREAD

**Discussion by joshnuss** [SvelteLab/SvelteLab#261] [Ideas]
Hi!

I was wondering if you would be open to adjusting some of the copy and hierarchy of the menu?

I think by simplifying the menus it could improve discoverability.

Here are a few improvements that can be made:

- Standardizing on verbs for `Add` / `New` / `Create`, and `Submit` / `Open`
- Adding a separate menu for Help could simplify the other menus (`Help -> SvelteKit Docs`, `Help -> SvelteLab Docs` & `Help -> Submit Issue`)
- Adding a separate `View` menu. Then `Toggle Terminal`, `Toggle File Tree`, become `View -> Terminal` & `View -> File Tree`
- Some menus use the term `Toggle x`, it could be better to change the labeling based on the state, for example, instead of a single `Toggle Config Files`, use two states `Show Config Files` & `Hide Config Files`  (maybe checkboxes in the menu would be helpful too)
- In some cases, we might be able to omit the word `Open`, for example `Open SvelteKit docs` could be simplified to just `SvelteKit Docs`
- Adding submenus, for example, instead of `Project > New Project` & `Project > New with Template`, have a submenu `Project -> New -> Blank` / `Project -> New -> From template`

I'd be happy to open a PR based on your feedback

**Comment by SarcevicAntonio** [comment]
Thanks for these awesome ideas! 👌

You can try opening a PR if you'd like, I'd really appreciate it 😁 not sure when I can review tho as I'm in the middle of moving 📦

Here are some of my thoughts:

• simplifying and standardizing the labeling would go a long way to make it feel cohesive. Just keep in mind that these are also used in the command palette so it should still also make sense and be searchable in both cases. Consider adding the `seo` prop to make searching commands easier. 

• changing labels to be dynamic could be tricky, but maybe you can find a way. Checkboxes would be sick! But this could also be done later in a separate step. 

• I'd try to keep the menus flat for now because I feel submenus could make stuff harder to find and access (more clicks), but maybe it works and is even better so feel free to try out, but I might ask to remove it again 😬😅😅

**Comment by joshnuss** [comment]
Thanks @SarcevicAntonio!

All good points.

What I can do is open a draft PR, and you guys can see if you like it. (no obligation to merge)

- For the command palette, it might make sense to have move commands that aren't on the menu. Like I think "Toggle X" is more handy for a command palette, by "Show X" / "Hide X" is better labeling for a menu. 
- For checkboxes, I'll see if it can be added. Though I wonder if the checkbox replaces the icon on the left? Or should it be on the right? I'll try
- For submenus, we can try it both ways!
 
I'll open a PR when I have some time

**Reply by SarcevicAntonio** [reply]
Sounds good! For the checkboxes I'd probably say replace the icons. Look what is possible and if you need pointers feel free to ask!
